compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -638,33 +636,38 @@ createProjectPackageDb opts fps = do
         -> Options
         -> FilePath
         -> FilePath
+        -> FilePath
         -> String
         -> String
         -> String
         -> Maybe String
         -> [String]
         -> IO ()
-    generateAndInstallIfaceFiles dalf src opts dbPathAbs projectPackageDatabaseAbs unitIdStr pkgIdStr pkgName mbPkgVersion deps = do
+    generateAndInstallIfaceFiles dalf src opts workDir dbPath projectPackageDatabase unitIdStr pkgIdStr pkgName mbPkgVersion deps = do
         loggerH <- getLogger opts "generate interface files"
-        mapM_ writeSrc src
+        let src' = [ (toNormalizedFilePath $ workDir </> fromNormalizedFilePath nfp, str) | (nfp, str) <- src]
+        mapM_ writeSrc src'
         opts' <-
             mkOptions $
             opts
-                { optWriteInterface = True
-                , optPackageDbs = projectPackageDatabaseAbs : optPackageDbs opts
-                , optIfaceDir = Just "./"
+                { optWriteInterface = False
+                , optPackageDbs = projectPackageDatabase : optPackageDbs opts
+                , optIfaceDir = Nothing
```

Comment:
Same question here, do we need to iface dir at all now?

Reply:
same as above.